tests/fixtures/settings.toml

```diff
@@ -0,0 +1,229 @@
+# This sample c2pa settings file enables a trust list. 
```

Comment:
I wonder if we should explain why TOML was picked as a format (I remember there were some challenges because of the type of embedded data. We may want to spare the gotcha to others?)